src/components/listBox/ListBoxReducers.ts

```diff
@@ -113,7 +91,6 @@ export const listBoxesReducer = (
         case ListBoxActions.select:
         case ListBoxActions.reorder:

```

Comment:
reorder ... is it still relevant, because there are no items to reorder anymore?

Reply:
its a simple reorder for the selected values. you can sent a new array with the selected values. It's used by the drag and drop